cmd/webhook/builder.go

```diff
@@ -7,15 +7,12 @@ import (
 	"github.com/Dynatrace/dynatrace-operator/cmd/config"
```

Comment:
is there any loss in functionality by no longer validating older versions? do we need more safety checks during conversion now?

Reply:
I think we need to test it, it's hard to say now :)